test/mlir/conversion/krnl_to_llvm/entry_point.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: onnx-mlir-opt --convert-krnl-to-llvm="use-opaque-pointers=true" --canonicalize %s -split-input-file | FileCheck %s
+// RUN: onnx-mlir-opt --canonicalize %s -split-input-file | FileCheck %s
```

Comment:
We have to keep `--convert-krnl-to-llvm` to lower krnl to llvm. So removing `="use-opaque-pointers=true"` should be enough.

Reply:
How did I not notice 🤦 Thank you very much for checking, I'll fix that right away